packages/status-js/src/client/client.ts

```diff
@@ -51,13 +56,21 @@ class Client {
 
   static async start(options: ClientOptions) {
     // Waku
+    const fleet =
+      options.environment === 'production'
+        ? discovery.predefined.Fleet.Prod
+        : discovery.predefined.Fleet.Test
+    /**
+     * >only connects to 1 remote node because of the limited number of nodes
+     * >run by Status and the limited number of connections provided by these nodes
+     * >
+     * >@see https://forum.vac.dev/t/waku-v2-scalability-studies/142/2
+     */
+    const peers = getPredefinedBootstrapNodes(fleet)
     const waku = await Waku.create({
       bootstrap: {
         default: false,
```

Comment:
Where does `default: true` connect?

Reply:
Connects to prod:

- https://github.com/status-im/js-waku/blob/6f5e1fa2aa2ec7b16be58e69f56efd775ca06554/src/lib/discovery/bootstrap.ts#L67
- https://github.com/status-im/js-waku/blob/6f5e1fa2aa2ec7b16be58e69f56efd775ca06554/src/lib/discovery/predefined.ts#L23

Reply:
Done at https://github.com/status-im/status-web/pull/294/commits/2bdc224320be9d5cf44f6cde0f62a70691101ca7